pkgs/development/python-modules/openstep-parser/default.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  pytestCheckHook,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "openstep-parser";
+  version = "2.0.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "kronenthaler";
+    repo = "openstep-parser";
+    rev = "refs/tags/${version}";
+    hash = "sha256-xBxbSKiF6zVTwZGGjPADx1LkhQjX2VHZsct/xf3rkTs=";
```

```suggestion
    hash = "sha256-saRf8vm1triAaJ+iZENtreb/QFUa3OICHh/7ImXA2yo=";
```
The hash seems be incorrect.
```
error: hash mismatch in fixed-output derivation '/nix/store/l185z4wnq6jllw78ga0n12v51z74xdga-source.drv':
         specified: sha256-xBxbSKiF6zVTwZGGjPADx1LkhQjX2VHZsct/xf3rkTs=
            got:    sha256-saRf8vm1triAaJ+iZENtreb/QFUa3OICHh/7ImXA2yo=
```

---

The correct hash is the old one in my machine. I tried to update with your suggestion but it is failing to build. How is it possible? Do i miss any step?

---

I noticed the error is only in linux but passed on darwin, should i make this darwin only package?

---

Same error after i change the hash https://github.com/NixOS/nixpkgs/pull/335100/checks?check_run_id=28900872656

---

I found test error on the original repo, is it because of that? Let me try to disable it now.

---

I tried to disable the failing check and changed the hash but still getting the error, do you know why?